cg/meta/upload/scout/raredisease_config_builder.py

```diff
@@ -63,8 +72,61 @@ def build_load_config(self) -> RarediseaseLoadConfig:
         self.include_pedigree_picture(load_config)
         load_config.custom_images = self.load_custom_image_sample()
         load_config.human_genome_build = GenomeBuild.hg19
+        load_config.rank_score_threshold = RANK_MODEL_THRESHOLD
+        load_config.rank_model_version = self.get_rank_model_version(variant_type=Variants.SNV)
+        load_config.sv_rank_model_version = self.get_rank_model_version(variant_type=Variants.SV)
         return load_config
 
+    def get_rank_model_version(self, variant_type) -> str:
+        hk_manifest_file: File = self.get_file_from_hk({HkNFAnalysisTags.MANIFEST})
+        if not hk_manifest_file:
+            raise FileNotFoundError("No manifest file found in housekeeper")
+        return self.extract_rank_model_from_manifest(hk_manifest_file, variant_type)
+
+    def extract_rank_model_from_manifest(self, hk_manifest_file, variant_type) -> str:
+        content: list[dict[str, str]] = ReadFile.get_content_from_file(
+            file_format=FileFormat.JSON, file_path=hk_manifest_file
+        )
+        return self.search_rank_model_in_manifest(content, variant_type)
+
+    def search_rank_model_in_manifest(self, content, variant_type) -> str:
+        """
+        Extract the version from manifest.json. Structure is
+        {pipeline, published, tasks}
+        content['tasks']: {{
+        'id': '6',
+        'name': 'NFCORE_RAREDISEASE:RAREDISEASE:PREPARE_REFERENCES:GET_CHROM_SIZES',
+        'cached': True,
+        'process': 'NFCORE_RAREDISEASE:RAREDISEASE:PREPARE_REFERENCES:GET_CHROM_SIZES',
+        'script': 'genmod \
+        score \
+        --rank_results \
+        --family_file cleanshrimp.ped \
+        --score_config rank_model_-v1.38-.ini \
+        --outfile cleanshrimp_snv_genmod_score_clinical_score.vcf \
+        cleanshrimp_snv_genmod_models_clinical_models.vcf
+
+        cat <<-END_VERSIONS > versions.yml
+        "NFCORE_RAREDISEASE:RAREDISEASE:RANK_VARIANTS_SNV:GENMOD_SCORE":
+            genmod: $(echo $(genmod --version 2>&1) | sed 's/^.*genmod version: //' )
+        END_VERSIONS"',
+        'inputs': [...],
+        'outputs': [...],
+        },
+        ...
+        }
+        """
+        pattern = variant_type.upper() + ":GENMOD_SCORE"
+        for key, value in content["tasks"].items():
+            process = value.get("process")  # Use get to safely access the 'process' key
+            script = value.get("script")
+            if pattern in process and AnalysisTag.CLINICAL in script:
+                match = re.search(r"v(\d+\.\d+)", script)  # Extract the version number
+                if match:
+                    version = match.group(1)
+                    return version
```

Comment:
This part has potential for dividing into another function
```suggestion
                return self._get_version_from_script(script)
```
being `_get_version_from_script`
```python
def _get_version_from_script(script: str) -> str:
    """<some_docstring>."""
    if match := re.search(r"v(\d+\.\d+)", script):
        return match.group(1)
    raise ValueError(...)
```

Reply:
maybe you don't need to raise an error, we can discuss later